services/email_verification.js

```diff
@@ -0,0 +1,79 @@
+/**
+* @file email_verification.js
+* @author Capuccino
+* @license BSD-3-Clause
+*/
+
+const micro = require("micro");
+const {json} = micro;
+const redis = require("redis");
+const Redite = require("redite");
+const config = require("./email.config");
+const snowflake = require("snowflake-codon");
+const client = redis.createClient({
+    host: config.redis.host,
+    password: config.redis.password
+});
+const db = new Redite({client});
```

Comment:
You import and create a Redite instance but never actually use it.

Reply:
I followed the Redite example so I wasn't sure to use it

Reply:
Well you don't actually use it at all, so best to remove it.